src/lib.rs

```diff
@@ -60,7 +64,7 @@ pub enum NotifyState<'a> {
     WatchdogTrigger,
     /// Resets the configured watchdog value.
     WatchdogUsec(u32),
-    /// Tells the service manager>  to extend the service timeout.
+    /// Tells the service manager to extend the service timeout.
```

Comment:
Sorry! I have a new keyboard and it's really sensitive, I've been making that mistake all week.